index.html

```diff
@@ -35,12 +39,199 @@ <h1 class="title">
+                        </button>
+                    </div>
+                </div>
+            </article>
+            <article class="portfolio">
+                <img src="./images/six.svg" alt="project one is a web site" class="project__photo">
+                <div class="project__info">
+                    <h3 class="project__title">Hello Microverse</h3>
+                    <div class="project__stack__container">
+                        <p class="project__stack">HTML/CSS</p>
+                        <p class="project__stack">Ruby on rails</p>
+                        <p class="project__stack">JavaScript</p>
+                    </div>
+                    <div class="project__button__holder">
+                        <button class="buttons project__button">
+                            See this project
+                            <img src="./images/Union.svg" alt="" class="arrow">
+                        </button>
+                    </div>
+                </div>
+            </article>
+        </section>
+
+        <section class="about">
+          <h2 class="h2">About</h2>
+          <p class="about__text">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod 
+          </p>
+          <button class="buttons button__getCV">
+            <img src="./images/downlolad CV.svg" alt="" class="downlaodIcon">
+          </button>
+
+          <div class="languages">
+            <h3 class="h3">Languages</h3>
+            <ul class="about__ul">
+                <li class="about__li">JavaScript(ES5/ES6)</li>
+                <li class="about__li">Ruby</li>
+                <li class="about__li">HTML</li>
+                <li class="about__li">CSS</li>
+            </ul>
+            <img src="./images/pounds.png" alt="" class="pound__logo">
+          </div>
+
+          <div class="frameworks">
+            <h3 class="h3">Frameworks</h3>
+            <ul class="about__ul">
+                <li class="about__li">Bootstrap</li>
+                <li class="about__li">Ruby on rails</li>
+                <li class="about__li">RSpec</li>
+                <li class="about__li">Capibara</li>
+                <li class="about__li">Selenium</li>
+            </ul>
+            
+            <img src="./images/Rectangle 19.svg" alt="" class="square__logo">
+          </div>
+
+          <div class="skills">
+            <h3 class="h3">Skills</h3>
+            <ul class="about__ul">
+                <li class="about__li">Codekit</li>
+                <li class="about__li">Codepen</li>
+                <li class="about__li">GitHub</li>
+                <li class="about__li">GitLab</li>
+                <li class="about__li">Terminal</li>
+            </ul>
+            <img src="./images/idea.png" alt="" class="idea__logo">
+          </div>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` 🙃 
- [ ] Then the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul`, and each of them a `li`. Please kindly fix it 😉 😉